chapters/playframework/index.md

```diff
@@ -0,0 +1,546 @@
+---
+layout: default
+project: playframework
+title: Play Framework
+chapter: true
+---
+
+![play_full_color](images/play.png)
+
+# Play Framework
```

Comment:
I'm not feeling very inspired right now. @michielha @rubenverboon @Vivami: any ideas?

Otherwise we'll just leave it like this now, and change the title if we think of something...